docroot/sites/dentistrycallctr/blt.yml

```diff
@@ -0,0 +1,16 @@
+project:
+  local:
+    hostname: null
+    protocol: null
+  machine_name: dentistrycallctr
+  human_name: dentistrycallctr
+drush:
+  aliases:
+    local: self
+    remote: dentistrycallctr.local
+drupal:
+  db:
+    database: sitenow
```

Comment:
Wrong database name. This probably taken from the default settings in blt/blt.yml.